### PR TITLE
Create categories components

### PR DESCRIPTION
**Pull request by JuanequeX**
Description
---
Create main component structure with styles, and create categories section.

### CODE REVIEW

File: app/javascript/components/sections/Section.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+const Section = (props) => {
+  return(
+    <a href={props.taxon.permalink} target="_blank" className='taxonomy'> 
+      <h1 className='taxonomy__title'>{props.taxon.name}</h1> 
+    </a>
+  );
+};
+
+export default Section;
```
Comment (OswaldoPineda) [inline app/javascript/components/sections/Section.js]: pls add an extra line here

File: app/javascript/components/sections/Sections.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <div className="cards-container">
+        {
+          taxonomies.map((taxonomy) => <Section taxon={taxonomy}/>)
+        }    
+      </div>  
+    </section>
+  );
+};
+
+export default Sections;
```
Comment (OswaldoPineda) [inline app/javascript/components/sections/Sections.js]: same here

File: app/javascript/components/sections/section.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    font-weight: 900;
+    line-height: 58px;  
+    text-align: center;
+
+    @media screen and (min-width: 700px) {
+      font-size: 24px;
+      line-height: 125px;
+    }
+  }
+}
```
Comment (OswaldoPineda) [inline app/javascript/components/sections/section.scss]: same

File: app/javascript/components/sections/sections.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    font-weight: 900;    
+    line-height: 1.2;
+  }
+
+  .cards-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+}
```
Comment (OswaldoPineda) [inline app/javascript/components/sections/sections.scss]: same

File: app/javascript/styles/_colors.scss [hunk 5]
```diff
@@ -0,0 +1 @@
+$white: #fff;
```
Comment (OswaldoPineda) [inline app/javascript/styles/_colors.scss]: same

File: app/javascript/styles/_fonts.scss [hunk 6]
```diff
@@ -0,0 +1,2 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap');
+$font-stack: 'Lato';
```
Comment (OswaldoPineda) [inline app/javascript/styles/_fonts.scss]: same

File: app/javascript/styles/_variables.scss [hunk 7]
```diff
@@ -0,0 +1 @@
+$lato-font: 'Lato';
```
Comment (OswaldoPineda) [inline app/javascript/styles/_variables.scss]: same

**Review by OswaldoPineda — CHANGES_REQUESTED** [review]
check the comments

File: app/javascript/components/sections/Section.js (hunk 1 above)
Comment (JuanequeX) [reply app/javascript/components/sections/Section.js]: done

File: app/javascript/components/sections/Sections.js (hunk 2 above)
Comment (JuanequeX) [reply app/javascript/components/sections/Sections.js]: Done

File: app/javascript/components/sections/section.scss (hunk 3 above)
Comment (JuanequeX) [reply app/javascript/components/sections/section.scss]: done

File: app/javascript/components/sections/sections.scss (hunk 4 above)
Comment (JuanequeX) [reply app/javascript/components/sections/sections.scss]: done

File: app/javascript/styles/_colors.scss (hunk 5 above)
Comment (JuanequeX) [reply app/javascript/styles/_colors.scss]: done

File: app/javascript/styles/_fonts.scss (hunk 6 above)
Comment (JuanequeX) [reply app/javascript/styles/_fonts.scss]: done

File: app/javascript/styles/_variables.scss (hunk 7 above)
Comment (JuanequeX) [reply app/javascript/styles/_variables.scss]: done

**Review by OswaldoPineda — APPROVED** [review]

**Pull request merged**